pydarn/plotting/fan.py

```diff
@@ -0,0 +1,246 @@
+# Copyright (C) 2020 SuperDARN Canada, University of Saskatchewan
+# Author: Daniel Billett, Marina Schmidt
+#
+# Modifications:
+#
+# Disclaimer:
+# pyDARN is under the LGPL v3 license found in the root directory LICENSE.md
+# Everyone is permitted to copy and distribute verbatim copies of this license
+# document, but changing it is not allowed.
+#
+# This version of the GNU Lesser General Public License incorporates the terms
+# and conditions of version 3 of the GNU General Public License, supplemented by
+# the additional permissions listed below.
+
+
+"""
+Fan plots, mapped to AACGM coordinates in a polar format
+"""
+
+import datetime as dt
+import matplotlib.pyplot as plt
+import numpy as np
+import os
+
+from matplotlib import ticker, cm, colors
+from typing import List
+
+# Third party libraries
+import aacgmv2
+
+from pydarn import PyDARNColormaps, build_scan, radar_fov
+
+class Fan():
+    """
+        Fan plots for SuperDARN data
+        This class inherits from matplotlib to generate the figures
+        Methods
+        -------
+        plot_fan
+        """
+
+    def __str__(self):
+        return "This class is static class that provides"\
+                " the following methods: \n"\
+                "   - plot_fan()\n"\
+                "   - return_beam_pos()\n"
+
+    @classmethod
+    def plot_fan(cls, dmap_data: List[dict], ax=None, scan_index: int = 1,
+                       ranges: List = [0,75], boundary: bool = True,
+                       parameter: str = 'v', lowlat: int = 50, cmap: str = None,
```

Comment:
```suggestion
                       parameter: str = 'v', lowlat: int = 30, cmap: str = None,
```
Again this is a suggestion because we have the option to change this with lowlat, I don't know if having it as 30 (which I think gets them all in) would be beneficial, would everyone using high lat radars then have to change it so their fov isn't tiny, I guess it's the opposite issue to the low lat radars getting cut off. The suggested change is there anyway!